jest.config.packages.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  collectCoverageFrom: ['./src/**/*.ts'],
+  collectCoverageFrom: ['./src/**/*.{ts,tsx}'],
```

Comment:
Updating global jest configurations for packages to collect testing coverage from `tsx` files